dlpy/model.py

```diff
@@ -24,6 +24,11 @@
 import pandas as pd
 import collections
 import sys
+import swat
+try:
```

Comment:
I am not happy the current design. Please create independent functions for this functionality and have everything in it. In this way, people without tf would also use dlpy.

Reply:
Moved import inside function that uses tf.